cadc-storage-permissions/README.md

```diff
@@ -1,2 +1,12 @@
 # storage-inventory
-Storage Inventory Permissions library
\ No newline at end of file
+Storage Inventory Permissions library
+
+## integration testing
+
```

Comment:
`baldur` is lower case only (everywhere)

Reply:
Updated.

Reply:
Fixed.